robottelo/hosts.py

```diff
@@ -1326,8 +1326,7 @@ def cli(self):
                         pass
         return self._cli
 
-    @property
-    def ui_session(self):
+    def ui_session(self, testname=None, user=None, password=None, url=None):
```

Comment:
@JacobCallahan I did not find it better to handle this. please let me know if you have anything.